### PR TITLE
[feat](Nereids) support refresh database command

### PR DESCRIPTION
**Pull request by vinlee19**
### What problem does this PR solve?

Issue Number: close #https://github.com/apache/doris/issues/42579

Related PR: #xxx

Problem Summary:
support refresh database command in Nereids.
### Release note

None

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by vinlee19** [comment]
run buildall

**Comment by vinlee19** [comment]
run cloud_p0

**Comment by vinlee19** [comment]
run buildall

**Comment by vinlee19** [comment]
run buildall

**Comment by LiBinfeng-01** [comment]
run p0

**Comment by vinlee19** [comment]
run buildall

**Review by LiBinfeng-01 — APPROVED** [review]

**Comment by LiBinfeng-01** [comment]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java [hunk 1]
```diff
@@ -4159,6 +4162,23 @@ public Object visitRefreshCatalog(RefreshCatalogContext ctx) {
     }
 
     @Override
+    public RefreshDatabaseCommand visitRefreshDatabase(RefreshDatabaseContext ctx) {
+        Map<String, String> properties = visitPropertyClause(ctx.propertyClause()) == null ? Maps.newHashMap()
+                : visitPropertyClause(ctx.propertyClause());
+        List<String> parts = visitMultipartIdentifier(ctx.name);
+        int size = parts.size();
+        Preconditions.checkArgument(size > 0, "database name can't be empty");
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java:4170]: throw new ParseException

File: fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java [hunk 2]
```diff
@@ -4159,6 +4162,23 @@ public Object visitRefreshCatalog(RefreshCatalogContext ctx) {
...
+        String dbName = parts.get(size - 1);
+
+        // [db].
+        if (size == 1) {
+            return new RefreshDatabaseCommand(dbName, properties);
+        } else if (parts.size() == 2) {  // [ctl,db].
+            return new RefreshDatabaseCommand(parts.get(0), dbName, properties);
+        }
+        throw new IllegalArgumentException("Only one dot can be in the name: " + String.join(".", parts));
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java:4179]: throw new ParseException

**Comment by vinlee19** [comment]
run buildall

**Comment by vinlee19** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Pull request merged**